js/modals.js

```diff
@@ -0,0 +1,98 @@
+const modals = [
+  {
+    title: 'Facebook 360',
+    desc: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea.",
+    langs: ['Html', 'Css', 'Javascript'],
+    imgCard: '.././assets/images/Portfolio1.png"',
```

Comment:
- [ ] Your images look broken because you have referenced the wrong path 😮 This issue is on all the project cards. Here is a screenshot that was taken 👇 

![image](https://user-images.githubusercontent.com/90222110/189144799-537862af-8e12-438a-a07b-3ea399da350c.png)

Fortunately, when I was trying to debug it, I found a way to resolve it. It is to add the correct path of each image and because all your images are located in the same folder, the path should look like the following
`./assets/images/{Image name}`.
⚠️  For the above image, please check the double quotes on the image path, I can see that there are two quotes at the end of the image path 😃 🏃 

HINT💡  For the above one, try `./assets/images/Portfolio1.png` and see how it looks like 😉

Reply:
well, that is weird, that didn't happen with anyone including my partner and two other reviewers.

Reply:
@BertrandConxy  can you please check now?